src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -38,6 +38,7 @@
   <ItemGroup Condition="'$(TargetGroup)'!='netstandard'">
     <Compile Include="System\BitConverterSpan.cs" />
     <Compile Include="System\BitConverter.netcoreapp.cs" />
+    <Compile Include="System\Convert.netcoreapp.cs" />
```

Comment:
NIT: This will also run on UWP but I guess we don't have a good naming conventions for non-netstandard stuff

Reply:
Yeah